README.md

```
@@ -117,6 +122,23 @@ When running in Docker, you need to tell the app how to translate paths between
+environment variables (useful for Docker deployments where you want secrets
+out of `config.json`):
+
+```yaml
+environment:
+  - JELLYFIN_API_KEY=abc123
+  - TMDB_API_KEY=xyz789
+  - TRAKT_CLIENT_ID=my-trakt-id
+```
+
+Environment variables take precedence over UI values and are **never** persisted
+back to `config.json`. See the [Environment Variables](#docker-environment-variables)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)